src/command/info/info.c

```diff
@@ -1295,20 +1297,19 @@ infoUpdateStanza(
                         infoPgCipherPass(infoBackupPg(stanzaRepo->repoList[repoIdx].backupInfo)));
                 }
 
-                // If a backup lock check has not already been performed, then do so
-                if (!stanzaRepo->backupLockChecked)
-                {
-                    // If there is a valid backup lock for this stanza then backup/expire must be running
-                    const LockReadResult lockResult = cmdLockRead(lockTypeBackup, stanzaRepo->name);
+                // If there is a valid backup lock for this stanza then backup/expire must be running
+                const LockReadResult lockResult = cmdLockRead(lockTypeBackup, stanzaRepo->name, repoIdx);
 
-                    stanzaRepo->backupLockHeld = lockResult.status == lockReadStatusValid;
-                    stanzaRepo->backupLockChecked = true;
+                if (lockResult.status == lockReadStatusValid)
+                {
+                    stanzaRepo->backupLockHeld = true;
 
-                    if (stanzaRepo->backupLockHeld)
+                    if (lockResult.data.size != NULL)
                     {
-                        stanzaRepo->percentComplete = lockResult.data.percentComplete;
-                        stanzaRepo->sizeComplete = lockResult.data.sizeComplete;
-                        stanzaRepo->size = lockResult.data.size;
+                        ASSERT(lockResult.data.size != NULL);
+
+                        stanzaRepo->sizeComplete += varUInt64(lockResult.data.sizeComplete);
+                        stanzaRepo->size += varUInt64(lockResult.data.size);
```

Comment:
These changed from '=' to '+=', do these run the risk of adding to random data in sizeComplete and size?

Reply:
We'll hope there is no random data in the lock file. But the change to += was intentional since there can be multiple backups running we add together to the progress so we maintain the current format. We'll likely break these out in the future -- see PR comment.